src/twisted/internet/base.py

```diff
@@ -719,14 +719,15 @@ def fireSystemEvent(self, eventType):
             event.fireEvent()
 
 
-    def addSystemEventTrigger(self, _phase, _eventType, _f, *args, **kw):
+    def addSystemEventTrigger(self, phase: str, eventType: str,
+                              f: Callable[..., Any], *args, **kw):
```

Comment:
This is still a signature change _f -> f 

But since _f is a "private" we can change it to public... but if it should be change, it should be change to `callable` to match the interface definition.

The interface is defined as
```
src/twisted/internet/interfaces.py:    def addSystemEventTrigger(phase, eventType, callable, *args, **kw):
```

Reply:
I took this signature change out